WordPress/src/main/res/layout/backup_list_subheader_item.xml

```diff
@@ -7,7 +7,7 @@
     android:layout_height="?attr/listPreferredItemHeight">
 
     <com.google.android.material.textview.MaterialTextView
-        android:id="@+id/backup_download_secondary_header"
+        android:id="@+id/text"
```

Comment:
Was it renamed to `text` to make it more generic and usable for the `restore` functionality? I know we're still finalising the names 😅,  just wanted to mention `text` looks too generic (if it is not already in your todo tasks list).

Reply:
It should be subheader 🤦 . I changed in commit 1009f5b